x-pack/plugin/security/build.gradle

```diff
@@ -306,6 +306,12 @@ test {
      * other if we allow them to set the number of available processors as it's set-once in Netty.
      */
     systemProperty 'es.set.netty.runtime.available.processors', 'false'
+
+    /*
+     * Some tests in this module set up a lot of transport threads so we reduce the buffer size per transport thread from the 1M default
+     * to keep direct memory usage under control.
+     */
+    systemProperty 'es.transport.buffer.size', '64k'
```

Comment:
Can we start at 256KB and see if it is still happening? That is what nio transport has and seems to never trigger this.

Reply:
Sure done :)